js/src/redux/providers/certifications/check-if-certified.js

```diff
@@ -0,0 +1,30 @@
+// Copyright 2015, 2016 Ethcore (UK) Ltd.
+// This file is part of Parity.
+
+// Parity is free software: you can redistribute it and/or modify
+// it under the terms of the GNU General Public License as published by
+// the Free Software Foundation, either version 3 of the License, or
+// (at your option) any later version.
+
+// Parity is distributed in the hope that it will be useful,
+// but WITHOUT ANY WARRANTY; without even the implied warranty of
+// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+// GNU General Public License for more details.
+
+// You should have received a copy of the GNU General Public License
+// along with Parity.  If not, see <http://www.gnu.org/licenses/>.
+
+import ABI from '../../../contracts/abi/certifier.json';
+
+export default (api) => {
```

Comment:
> Any specific reason why we are in a "only a single function per file" mode here?

This is just personal taste & being used to it. Will remove as far as I agree with the following point.

> I would put all files in this directory into one, easy to access, easy to see the whole picture

On purpose, I stored middleware, actions and reducers in separate files. It underlines the whole style of using Redux by making it less tempting to intertwine actions & their reactions.

What about moving `checkIfCertified` and `fetchCertifier` into `middleware.js`?